inlong-audit/audit-common/src/main/java/org/apache/inlong/audit/protocol/AuditApi.java

```diff
@@ -0,0 +1,6844 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// source: AuditApi.proto
+
+package org.apache.inlong.audit.protocol;
```

Comment:
Do we really need to upload this generated file?

Reply:
yes

Reply:
without this class file ，will get build error

Reply:
Why？ I think with the  .proto definition file and maven plugin commit the .proto file is enough, we can generate the target file locally

Reply:
has delete protoc java api file ,and add build plugin in config in pom.xml